src/Maker/MakeResetPassword.php

```diff
@@ -0,0 +1,277 @@
+use Symfony\Bundle\MakerBundle\Generator;
+use Symfony\Bundle\MakerBundle\InputConfiguration;
+use Symfony\Bundle\MakerBundle\Security\InteractiveSecurityHelper;
+use Symfony\Bundle\MakerBundle\Util\YamlSourceManipulator;
+use Symfony\Bundle\MakerBundle\Validator;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use SymfonyCasts\Bundle\ResetPassword\SymfonyCastsResetPasswordBundle;
+
+class MakeResetPassword extends AbstractMaker
+{
+    /**
+     * @var FileManager
+     */
+    private $fileManager;
+
+    public function __construct(FileManager $fileManager)
+    {
+        $this->fileManager = $fileManager;
+    }
+
+    public static function getCommandName(): string
+    {
+        return 'make:reset-password';
+    }
+
+    public function configureCommand(
+        Command $command,
+        InputConfiguration $inputConfig
+    ) {
+        $command
+            ->setDescription('Create controller, entity, and repositories for use with SymfonyCasts Reset Password Bundle.')
+        ;
+    }
+
+    public function configureDependencies(DependencyBuilder $dependencies)
+    {
+        $dependencies->addClassDependency(SymfonyCastsResetPasswordBundle::class, 'symfonycasts/reset-password-bundle');
+    }
+
+    public function interact(InputInterface $input, ConsoleStyle $io, Command $command)
+    {
+        $io->title('Reset Password Bundle Requires:');
+        $requirements[] = '1) A user entity has been created.';
+        $requirements[] = '2) The user entity contains an email property with a getter method.';
+        $requirements[] = '3) A user repository exists for the user entity.'."\n";
+        $requirements[] = '<fg=yellow>bin/console make:user</> will generate the user entity and it\'s repository...'."\n";
+        $io->text($requirements);
+
+        // initialize arguments & commands that are internal (i.e. meant only to be asked)
+        $command
+            ->addArgument('from-email-address', InputArgument::REQUIRED)
+            ->addArgument('from-email-name', InputArgument::REQUIRED)
+            ->addArgument('controller-reset-success-redirect', InputArgument::REQUIRED)
+            ->addArgument('user-class')
+            ->addArgument('email-property-name')
+            ->addArgument('email-getter')
+            ->addArgument('password-setter')
+        ;
+
+        $io->section('- Email Templates -');
+        $emailText[] = 'Please answer the following questions that will be used to generate the email templates.';
+        $emailText[] = 'If you are unsure of what these answers should be, that\'s ok.';
+        $emailText[] = 'You can change these later after the templates have been generated.';
+        $io->text($emailText);
+
+        $input->setArgument('from-email-address', $io->ask(
+            'What email address will be used to send reset confirmations? I.e. admin@your-domain.com',
```

Comment:
Use e.g. - there's a subtle difference between e.g. and i.e. ... and I think e.g. is the correct one

Reply:
done